pallets/proxy-keystore/src/tests.rs

```diff
@@ -0,0 +1,655 @@
+use super::*;
+use crate::mock::*;
+use crate::Event as CrateEvent;
+use crate::mock::Event as MockEvent;
+use frame_support::{assert_err, assert_ok};
+use frame_system::{Account, AccountInfo};
+use pallet_balances::AccountData;
+use sp_runtime::testing::H256;
+
+#[test]
+fn create_keystore() {
+    new_test_ext().execute_with(|| {
+        let keys = get_test_keys();
+
```

Comment:
```suggestion
```

Reply:
I might remove this completely, gonna fix if not.